Ryujinx/Ui/GLScreen.cs

```
@@ -251,375 +252,42 @@ private new void UpdateFrame()
-                                HidControllerLayouts.Joined,
-                                CurrentButtonsKeyboard,
-                                LeftJoystickKeyboard,
-                                RightJoystickKeyboard);
-                            Ns.Hid.SetJoyconButton(
-                                HidControllerId.CONTROLLER_UNKNOWN,
-                                HidControllerLayouts.Main,
-                                CurrentButtonsKeyboard,
-                                LeftJoystickKeyboard,
-                                RightJoystickKeyboard);
-                            break;
-                        default:
-                            if (HidEmulatedDevices.Devices.PlayerUnknown < 0)
-                                throw new ArgumentException("Unknown Emulated Device Code: " + HidEmulatedDevices.Devices.PlayerUnknown + ".");
-                            Ns.Hid.SetJoyconButton(
-                                HidControllerId.CONTROLLER_UNKNOWN,
-                                HidControllerLayouts.Joined,
-                                CurrentButtonsGamePad[HidEmulatedDevices.Devices.PlayerUnknown],
-                                LeftJoystickGamePad[HidEmulatedDevices.Devices.PlayerUnknown],
-                                RightJoystickGamePad[HidEmulatedDevices.Devices.PlayerUnknown]);
-                            Ns.Hid.SetJoyconButton(
-                                HidControllerId.CONTROLLER_UNKNOWN,
-                                HidControllerLayouts.Main,
-                                CurrentButtonsGamePad[HidEmulatedDevices.Devices.PlayerUnknown],
-                                LeftJoystickGamePad[HidEmulatedDevices.Devices.PlayerUnknown],
-                                RightJoystickGamePad[HidEmulatedDevices.Devices.PlayerUnknown]);
-                            break;
-                    }
-                }
-            }
+            return -1;
```

nit: It'd be better to throw an ArgumentException if it's the argument is invalid. Returning `-1` will crash on unrelated code.